packages/tools/stylelint-no-logical-props-shorthands/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@justeattakeaway/stylelint-no-logical-props-shorthands",
+  "version": "0.0.1",
+  "description": "Stylelint plugin for disallowing logical props shorthands.",
+  "license": "Apache-2.0",
```

Comment:
Just make sure to add a `"main": "index.js"` entry here, so that the file is exposed as part of the published package when consumed outside the monorepo 👍🏼